sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala

```diff
@@ -181,6 +181,12 @@ object GpuFilter extends Arm {
 case class GpuFilterExec(condition: Expression, child: SparkPlan)
     extends UnaryExecNode with GpuPredicateHelper with GpuExec {
 
+  // we do not record totalTime for this operator
```

Comment:
Similar [comment](https://github.com/NVIDIA/spark-rapids/pull/2462#discussion_r636961988) as I made in #2462 about refactoring this for reuse.  May be simpler to just do them both in the same PR since the change is so small.